common/types/shared.go

```diff
@@ -252,7 +252,7 @@ func (v *ActivityTaskFailedEventAttributes) GetIdentity() (o string) {
 type ActivityTaskScheduledEventAttributes struct {
 	ActivityID                    string        `json:"activityId,omitempty"`
 	ActivityType                  *ActivityType `json:"activityType,omitempty"`
-	Domain                        string        `json:"domain,omitempty"`
+	Domain                        *string       `json:"domain,omitempty"`
```

Comment:
does the idl needs to be updated as well?

Reply:
thrift generated files always have pointer. We only dropped pointer for internal types I think.